CHANGELOG.md

```diff
@@ -8,6 +8,9 @@ MOVE THIS SECTION TO THE CORRECT LOCATION
 
 -Features for next release
 
+- **AWSIoT**
+  - feat(AWSIoT): adds back Message type ([PR #3852](https://github.com/aws-amplify/aws-sdk-ios/pull/3852))
```

Comment:
shold this be a fix?

Reply:
I looked and did not see it breaking without this header included. The individual header is public so it is included as a public header in the framework so it has always been available since it was added. I believe this is not as much a fix as it is exposing a public type. Since the integration tests did not fail without this header I think we can call this one `feat` since it is exposing the type for the feature it supports.

Reply:
Just tried it from Swift. It breaks the build if you use the subscribe method which needs this type. I will change to `fix` and update CHANGELOG.md. On the Objective-C side it can build and run all tests without a problem.